src/tools/spin/spinquic.cpp

```diff
@@ -554,8 +556,14 @@ void Spin(LockableVector<HQUIC>& Connections, std::vector<HQUIC>* Listeners = nu
                 auto Stream = ctx->TryGetStream();
                 if (Stream == nullptr) continue;
                 auto BytesRemaining = MsQuic->GetContext(Stream);
-                MsQuic->StreamReceiveComplete(Stream, (uint64_t)BytesRemaining);
-                MsQuic->SetContext(Stream, nullptr);
+                if (BytesRemaining != nullptr && GetRandom(10) == 0) {
+                    auto BytesConsumed = GetRandom((size_t)BytesRemaining);
```

Comment:
```suggestion
                    auto BytesConsumed = GetRandom((uint64_t)BytesRemaining);
```
This is better. It's still uint64_t on 32-bit as well.